proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -377,3 +378,12 @@ message Telemetry {
   // See https://github.com/open-telemetry/opentelemetry-go/tree/main/metric
   google.protobuf.BoolValue metrics_enabled = 2;
 }
+
+message MetricsExporterConfig {
+  // when `true` metrics will be exported as spans from libtraceable
+  google.protobuf.BoolValue enabled = 1;
+  // max interval for calls to TPA
+  google.protobuf.Int32Value export_interval_ms  = 2;
```

Comment:
also we dont actually need to expose these 2 configs as well. I exposed them in libtracaeble because that'll let us avoid a code change in libtraceable in future if we decide to change it. But in goagent or any other agents, we can hardcode the otel defaults